common/model/opening-hours.js

```diff
@@ -39,93 +40,36 @@ export type ExceptionalVenueHours = {|
   exceptionalDay: Days,
   id: string,
   name: string,
+  order: number,
   openingHours: OpeningHoursDay | ExceptionalOpeningHoursDay,
   opensChanged: boolean,
   closesChanged: boolean
 |};
 
-export type PlacesOpeningHours = Array<Place>;
+export type PlacesOpeningHours = Array<Venue>;
```

Comment:
I might add it to the linter... @davidpmccormick are we deciding to use `[]` and `{}`?

Reply:
Happy for it to be added to the linter, yeah.